automated/linux/device-tree/device-tree.sh

```diff
@@ -1,75 +1,47 @@
 #!/bin/sh
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2016-2020 Linaro Ltd.
```

Comment:
all files in this repository are GPL2-only by default: https://github.com/Linaro/test-definitions/blob/master/COPYRIGHTS#L14. So this bit isn't required IMHO. If you would like to change the license, you need to add GPL-2.0-or-later to LICENSES directory.

Reply:
I only replaced the license text by the SPDX identifier. The license on this file is GPL2+. I'm fine to relicense the code to GPL2 only but it requires its authors to agree. So adding GPL-2.0-or-later to LICENSE doesn't mean change the license, but it means keep the file licensed GPL2+.

Reply:
Either fix will do. I'm OK with relicensing to GPL2 (IIRC I asked authors a long time ago and there was no objections) or keeping GPL2+. License check needs the license file in the repository, that's why it's failing. You can also list it in a different section in COPYRIGHTS file without adding SPDX header.

Reply:
sounds good. I'll relicense to gpl2 only then. Thanks.